manifests/init.pp

```diff
@@ -102,6 +102,7 @@
   }
 
   $nss_db_dir   = "${pki_dir}/nssdb"
+  $nss_db_password_file = "${pki_dir}/nss_db_password-file"
```

Comment:
What is the benefit of having this in `init.pp`?

Reply:
I thought puppet-katello was pulling it from here, but on a closer reading it actually references `certs::qpid::nss_db_password_file` which in turn pulls it from `certs::ssltools::nssdb::nss_db_password_file` so I think we're good to move it to nssdb.pp